lib/plugins/aws/package/compile/events/apiGateway/lib/hack/updateStage.js

```diff
@@ -118,7 +118,8 @@ function resolveApiGatewayResource(resources) {
   if (apiGatewayResourcesIds.length !== 1) return null;
   const apiGatewayResourceId = apiGatewayResourcesIds[0];
   if (
-    !_.findKey(resources, resource => {
+    !Object.keys(resources).find(key => {
```

Comment:
I think more natural would be to use `some` here (`find` is good when we're interested in _key_ value, and not just fact that it matches some conditions)